canonical/react-components
# Update Storybook to use webpack v5

## Done

- Updates Storybook config to use Webpack v5
- Upgrades css-loader and style-loader to latest versions (that require webpack v5)
- Drive-by: fix failing focus test in navigation component

## QA

- Open demo: https://react-components-809.demos.haus/
- Make sure Storybook renders as expected
- Make sure "Docs" tab is present for components

## Fixes

Fixes: #670 
Fixes: #650 #662
